onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -1031,7 +1017,29 @@ Applies to session load, initialization, etc. Default is 0.)pbdoc")
           },
           R"pbdoc(Graph optimization level for this session.)pbdoc")
       .def_readwrite("use_deterministic_compute", &SessionOptions::use_deterministic_compute,
-                     R"pbdoc(Whether to use deterministic compute. Default is false.)pbdoc");
+                     R"pbdoc(Whether to use deterministic compute. Default is false.)pbdoc")
+      .def(
```

Comment:
The is the real change. Three new APIs have been added. The rest is just formatting related changes.